src/backend/app/email_templates/mapping/requests.html

```diff
@@ -130,7 +130,7 @@ <h2>Mapping Task Details</h2>
             <p><strong>Project:</strong>{{project_name}}</p>
             <p><strong>Description:</strong> {{description}}</p>
           </div>
-          <a href="dronetm.naxa.com.np" class="task-button"
+          <a href="https://dronetm.org/" class="task-button"
```

Comment:
We should take a base url from env.

Reply:
> We should take a base url from env.

@nrjadkry Good point! I've updated the code to load the base URL from the environment variables.